packages/typespec-autorest/test/decorators.test.ts

```diff
@@ -287,7 +287,7 @@ describe("typespec-autorest: decorators", () => {
     it("duplicate example file", async () => {
       const host = await createAutorestTestHost();
       const runner = await createAutorestTestRunner(host, {
-        "examples-directory": resolveVirtualPath("./examples"),
+        "examples-dir": resolveVirtualPath("./examples"),
```

Comment:
Seems like we should add a test dir with no configuration setting for examples to protect the default behavior (loading from "./examples") - we could duplicate this test with no examples-dir setting, for example

Reply:
added more test